packages/scaffolding-cli/templates/src/ssr/package-lock.json

```diff
@@ -14838,66 +15285,74 @@
       "integrity": "sha512-iyam8fBuCUpWeKPGpaNMetEocMt364qkCsfL9JuhjXX6dRnguRVOfk2GZaDpPjcOKiiXCPINZC1GczQ7iTq3Zw=="
     },
     "next": {
-      "version": "9.3.5",
-      "resolved": "https://registry.npmjs.org/next/-/next-9.3.5.tgz",
-      "integrity": "sha512-vJKGUK2nMsveYZTZH5Oz9L+cDYcCsuWLUXfWJejvhdQ3jLgVhZVa+7TQ45Kt0PA1ea7jqlDUN22DtzmPFi981w==",
-      "requires": {
-        "@ampproject/toolbox-optimizer": "2.2.0",
-        "@babel/core": "7.7.2",
-        "@babel/plugin-proposal-class-properties": "7.7.0",
-        "@babel/plugin-proposal-nullish-coalescing-operator": "7.7.4",
+      "version": "9.4.1",
```

Comment:
Do we need to pin this version?

Reply:
I reckon we leave with ~9.4 as this is a new release. we will expect some patches. What do you say? @estherthetesteramido